events_test/src/test/java/com/adobe/aio/event/util/DataNodeUtil.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2017 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package com.adobe.aio.event.util;
+
+public class DataNodeUtil {
+
+  private static final String DATA_EVENT_ID_NODE = "data_event_id";
+
+  private DataNodeUtil() {
+    throw new IllegalStateException("Should not to be instantiated");
+  }
+
+  public static String getEventDataNode(String eventId) {
```

Comment:
wdyt moving this method within `PublishServiceIntegrationTest` ?

Reply:
ok

Reply:
it is getting used in the `JournalServiceIntegrationTest` also.

Reply:
Let's keep it here only.